docs/sections/user_guide/rocoto_yaml.rst

```diff
@@ -0,0 +1,604 @@
+    FOO: 12
+    BAR: baz
+
+.. code:: XML
+
+  <?xml version="1.0"?>
+  <!DOCTYPE workflow
+  [
+      <!ENTITY FOO "12">
+      <!ENTITY BAR "baz">
+  ]>
+
+Defining the workflow log
+_________________________
+
+``log:`` is a required entry.
+
+
+.. code::
+
+  log: /some/path/to/workflow.log
+
+.. code:: XML
+
+  <log>/some/path/to/workflow.log</log>
+
+A cycle string may be specified here, instead.
+
+.. code::
+
+  log:
+    cyclestr:
+      value: /some/path/to/workflow_@Y@m@d.log
+
+.. code:: XML
+
+  <log><cyclestr>/some/path/to/workflow_@Y@m@d.log</cyclestr></log>
+
+
+Defining the set of tasks
+_________________________
+
+At least one task or metatask must be defined in the ``tasks:`` section.
+
+.. code::
+
+  tasks:
+    task_*:
+    metatask_*
+
+
+The ``<task>`` tag
+..................
+
+Multiple ``task_*:`` YAML entries may exist under the ``dependency:`` entry, or any of the
+``metatask_*:`` entries. At least one must be specified per workflow.
+
+.. code::
+
+  task_foo:
+    attrs:
+      cycledefs: hourly
+      maxtries: 2
+      throttle: 10
+      final: false
+    command: echo hello world
+    walltime: 00:10:00
+    cores: 1
+
+
+.. code::
+
+  <task name="foo" cycledefs:"hourly" maxtries="2" throttle="10" final="False">
+    ...
+  </task>
+
+
+The following entries take strings just like in the ``command:`` example above. Please see the :rocoto:`Rocoto documentation<>` for specifics on how to set them.
+
+.. code::
+
+  account:
+  exclusive:
+  jobname:
+  join:
+  memory:
+  native:
+  nodes:
+  partition:
+  queue:
+  rewind:
+  shared:
+  stderr:
+  stdout:
+
+The following UW YAML entries require values that are integers, strings, or ``cyclestr:`` entries.
+
+.. code::
+
+  command:
+  deadline:
+  jobname:
+  join:
+  native:
+  stderr:
+  stdout:
+
+The ``<dependency>`` tag
+........................
+
+The ``<dependency>`` tag has many different tags for defining the readiness of a task to run. They may be categorized in several ways: boolean operator tags, comparison tags, and dependencies.
+
+Boolean Operator Tags
+_____________________
+
+All of the boolean operator tags require **one or more additional dependency tags** from any category in the sub-tree of the entry.
+
+.. code:: text
+
+  and:
+  or:
+  not:
+  nand:
+  nor:
+  xor:
+  some:
+
+Comparison Tags
+_______________
+
+Both the ``streq:`` and ``strneq:`` UW YAML entries are specified the same way. The sub-tree ``left:`` and ``right:`` entries both accept a ``cyclestr:`` if needed.
+
+.. code::
+
+  streq:
+    left: &FOO;
+    right: bar
+
+.. code:: XML
+
+  <dependency>
+    <streq>
+      <left>&FOO;</left>
+      <right>bar</right>
+    </streq>
+  </dependency>
+
+Dependency tags
+_______________
+
+These tags define dependencies on other tasks, metatasks, data, or wall time.
+
+* The task dependency
+
+.. code::
+
+  taskdep:
+    attrs:
+      cycle_offset: "-06:00:00"
+      state: succeeded
+      task: hello                # required
```

Comment:
```suggestion
      task: hello # required
```
There's some variety in the number of spaces before the YAML comments. Is there a pattern here I'm not seeing? Were you just trying to get out to the right of the text block for visibility? IMO the syntax highlighting, especially if we mark YAML blocks as `code-block:: yaml`, is good enough that we could potentially just follow the Python advice and not try to align these.